src/Microsoft.DotNet.Build.Tasks/PackageFiles/test-runtime/project.json

```diff
@@ -1,33 +0,0 @@
-{
-  "dependencies": {
```

Comment:
I am not aware of WCF using it from the build tools package, I think we do the same as CoreFx.